_extras/checkout.md

```diff
@@ -219,13 +219,14 @@ think your lesson has good start points for participatory instruction, you may e
 work for a checkout demo. Be sure to give yourself time to change course if your planned lesson will not work.
 
 ### Sign-up and Set-up
-To sign up, select a session that works for you on the [the teaching demo schedule Etherpad]({{page.demopad}}), and add
-your name and a link to your lesson of choice to the Etherpad. Be sure to **double check the time in your local time zone** by clicking on the converter
+To sign up, select a session that works for you on [the Training Demonstration Sessions Etherpad]({{page.demopad}}), and add
+your name and a link to your lesson of choice to that Etherpad. Be sure to **double check the time in your local time zone** by clicking on the converter
 link posted. Also, examine the demo description to ensure that it is not a special session targeting a specific sub-community or
 language (unless you are part of that target group).
 
-The link to connect to the video conference is at the top of the Etherpad. Your Trainer may send an email with confirmation and/or
-additional instructions prior to your demo. If possible, plan to use a headset with built in microphone for the best audio experience.
+The link to connect to the video conference is at the top of the Traning Demonstration Sessions Etherpad. 
```

Comment:
```suggestion
The link to connect to the video conference is at the top of the Teaching Demos Etherpad. 
```
The word teaching missing from the title was misleading, I've corrected it on the etherpad and this should match

Reply:
Re line 227, the current title of the document is "The Carpentries Instructor Training Teaching Demonstration Sessions". "Teaching Demo Etherpad" is confusing until that becomes the title of the document.